lib/child_process.js

```diff
@@ -98,7 +98,7 @@ const MAX_BUFFER = 1024 * 1024;
  * @param {{
  *   cwd?: string;
  *   detached?: boolean;
- *   env?: object;
+ *   env?: object<string, string>;
```

Comment:
```suggestion
 *   env?: Record<string, string>;
```

Reply:
That's TypeScript, but it's not JSDoc, is it? I can only find it in places like https://www.typescriptlang.org/docs/handbook/utility-types.html#recordkeys-type but don't see it in https://jsdoc.app/ or even [TypeScript's documentation about JSDoc](https://www.typescriptlang.org/docs/handbook/jsdoc-supported-types.html).

Reply:
`object<string, string>` is probably not JSDoc either, at least my code editor doesn't understand it.

![screenshot of VSCode not understanding `object<string,string>` syntax in JSDoc](https://user-images.githubusercontent.com/14309773/160380636-228f696a-048b-4af0-870e-457f07d00162.png)

![screenshot of VSCode understanding `Record<string,string>` syntax in JSDoc](https://user-images.githubusercontent.com/14309773/160380679-bb0baf1c-deab-49b4-b21e-bdf1391e685c.png)

 I'm tempted to use TS syntax as long as it works, unless there's a reason not to?

Reply:
> `object<string, string>` is probably not JSDoc either, at least my code editor doesn't understand it.

https://jsdoc.app/tags-type.html shows the syntax. Its example is `Object<string, number>`.

Reply:
> I'm tempted to use TS syntax as long as it works, unless there's a reason not to?

Yeah, we can do that and always change it if it turns out to be less compatible somehow or whatever.